obal/data/module_utils/copr.py

```diff
@@ -0,0 +1,29 @@
+"""
+A copr-cli wrapper and support functions for Copr
+"""
+from subprocess import check_output, CalledProcessError
+
+class CoprCliCommandError(Exception):
+    """Raised when copr-cli command fails"""
+    def __init__(self, message, command):
+        self.message = message
+        self.command = command
+        super(CoprCliCommandError, self).__init__(message) #pylint: disable-all
+
+def copr_cli(command, executable=None):
+    """
+    Run a copr-cli command
+    """
+    if executable is None:
+        executable = 'copr-cli'
+
+    try:
+        return check_output([executable] + command, universal_newlines=True)
```

Comment:
Check output only captures stdout by default.

You gotta capture stderr too, like we do in
https://github.com/theforeman/obal/blob/60e8b53f1993e961f9f26a922633194648a0184a/obal/data/module_utils/obal.py#L112

See also https://docs.python.org/3/library/subprocess.html#subprocess.check_output

Reply:
Ha! What timing, I updated this.